SWLOR.Game.Server/Feature/ChatCommandDefinition/DMChatCommand.cs

```diff
@@ -845,5 +851,33 @@ private void CreatureManager()
                     Gui.TogglePlayerWindow(user, GuiWindowType.CreatureManager);
                 });
         }
+        private void DMBroadcast()
+        {
+            _builder.Create("bc")
+                .Description("Sends your DM shout to Discord.")
+                .Permissions(AuthorizationLevel.DM, AuthorizationLevel.Admin)
```

Comment:
You need to add validation check to this definition in the event the DM forgets to send a message.


Example:
```

                .Validate((user, args) =>
                {
                    if (args.Length <= 0)
                        return "Please enter a message.";

                    return string.Empty;
                })
```

Reply:
Added